marlowe-dashboard-client/src/Pickup/State.purs

```diff
@@ -59,53 +64,98 @@ handleAction ::
   Action -> HalogenM State Action ChildSlots Msg m Unit
 handleAction (OpenCard card) = assign _card $ Just card
 
-handleAction CloseCard = assign _card Nothing
+handleAction CloseCard = do
+  assign _card Nothing
+  assign _remoteWalletDetails NotAsked
```

Comment:
you could merge this assign into a single modify

Reply:
Good catch. Although, looking at this fresh in the morning, I'm not sure I need to reset remoteWalletData here anyway. Either way I'll change it in part 2 - not worth waiting for hydra for this.